src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2223,45 +2282,74 @@
         <Setter Property="FontSize" Value="14" />
     </Style>
 
-    <Style x:Key="CaptionButtonStyleClose" TargetType="Button">
+    <Style x:Key="CloseButtonStyle" TargetType="Button">
+        <Setter Property="Cursor" Value="Hand"/>
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="Button">
-                    <Grid x:Name="LayoutRoot"
-                          Width="46"
-                          Height="40"
-                          Background="Transparent">
-                        <TextBlock x:Name="txt"
-                                   HorizontalAlignment="Center"
-                                   VerticalAlignment="Center"
-                                   FontFamily="Segoe MDL2 Assets"
-                                   FontSize="14"
-                                   Foreground="{StaticResource PreferencesWindowFontColor}"
-                                   RenderOptions.ClearTypeHint="Auto"
-                                   Text="{TemplateBinding Content}"
-                                   TextOptions.TextFormattingMode="Display"
-                                   TextOptions.TextRenderingMode="Aliased" />
+                    <Grid>
+                        <Image Name="closeIcon"  Width="16" Height="16" Source="pack://application:,,,/DynamoCoreWpf;component/UI/Images/close_grey.png"/>
                     </Grid>
                     <ControlTemplate.Triggers>
                         <Trigger Property="IsMouseOver" Value="True">
-                            <Setter TargetName="LayoutRoot" Property="Background" Value="{StaticResource CloseButtonMouseOverBackground}" />
-                            <Setter TargetName="txt" Property="Foreground" Value="{StaticResource CloseButtonMouseOverForeground}" />
-                        </Trigger>
-                        <Trigger Property="IsPressed" Value="True">
-                            <Setter TargetName="LayoutRoot" Property="Background" Value="{StaticResource CloseButtonPressed}" />
-                            <Setter TargetName="txt" Property="Foreground" Value="{StaticResource CloseButtonMouseOverForeground}" />
-                            <Setter TargetName="txt" Property="FontSize" Value="12" />
+                            <Setter TargetName="closeIcon" Property="Source" Value="pack://application:,,,/DynamoCoreWpf;component/UI/Images/close_blue.png" />
                         </Trigger>
                     </ControlTemplate.Triggers>
                 </ControlTemplate>
             </Setter.Value>
         </Setter>
     </Style>
 
-    <Style x:Key="CloseButtonStyle"
-           BasedOn="{StaticResource CaptionButtonStyleClose}"
-           TargetType="Button">
-        <Setter Property="Content" Value="&#xE8BB;" />
+    <Style x:Key="CtaButtonStyle" TargetType="Button">
```

Comment:
I dont think you need to rename this style since it is used in other places, and our team meant to apply this universally